File: 01 Introdução/1046 Tempo de Jogo/1046.java
import java.util.Scanner;

class Main {

  public static void main(String args[]) {
    Scanner sc = new Scanner(System.in);

    int begin = sc.nextInt();
    int end = sc.nextInt();

    if (end > begin) {
      System.out.printf("O JOGO DUROU %d HORA(S)\n", end - begin);
    } else {
      System.out.printf("O JOGO DUROU %d HORA(S)\n", 24 - begin + end);
    }

    sc.close();
  }
}

// https://www.beecrowd.com.br/judge/pt/problems/view/1046
// Ranking 08670º, Time 0.272, Submiss Date 12/02/2023 23:56:12 (Java 14)

File: 01 Introdução/1046 Tempo de Jogo/1046.py
begin, end = [int(x) for x in input().split(' ')]

if end > begin:
    print(f'O JOGO DUROU {end - begin} HORA(S)')
else:
    print(f'O JOGO DUROU {24 - begin + end} HORA(S)')

# https://www.beecrowd.com.br/judge/pt/problems/view/1046
# Ranking 01462º, Time 0.000, Submiss Date 02/03/2023 20:14:05 (Python 3.9)
